web/cypress/integration/route/search-route.spec.js

```diff
@@ -89,8 +89,8 @@ context('Create and Search Route', () => {
     cy.contains('Route').click();
 
     // search one label
-    cy.get('[title=Labels]').click();
     cy.wait(500);
+    cy.get('[title=Labels]').click();
```

Comment:
`domeSelector`

Reply:
This is a test under route, we will change these specifications in the new PR.

Reply:
not conflicted to add this to `domSelector`